src/main/java/com/google/sps/servlets/Search.java

```diff
@@ -0,0 +1,67 @@
+package com.google.sps.servlets;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.*;
+import java.util.*;
+import java.util.Enumeration;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+@WebServlet("/search")
+public class Search extends HttpServlet {
+      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+
+        //CASE 1: RESPONSE RETURNED AS TITLE CATEGORY
+        System.out.println("this is post request");
+        String returnedValue = request.getParameter("searchBar");
+        final String entity_questionDB = "survey";
+        System.out.println("Returned value is " + returnedValue);
+
+
+        //If is question TITLE
+        final String questionTitle = "question";
+        Query query = new Query(entity_questionDB).addFilter(questionTitle,FilterOperator.EQUAL,returnedValue);
```

Comment:
i don't know if we want to go with EQUAL, because its not a good way of searching. For example, if i search for movies with the intention of getting "what movie should i watch today?", then I wont see anything then. However, maybe using a CONTAINS instead would work??

Reply:
Please, try to make the match more inclusive? for example if the query is "what movie should i watch today"

replace trivial words like "i", "what", "should", and maybe just keep important words like"movie", "today" and match it with any other question that contains these improtant terms.
for trivial words, we can create a manual list of these trivial words..